packages/components/MenuButton/src/MenuButton.tsx

```diff
@@ -92,7 +93,10 @@ export const MenuButton = compose<MenuButtonType>({
     contextualMenu: { slotType: ContextualMenu as React.ComponentType<object> },
     contextualMenuItems: React.Fragment,
   },
-  styles: {},
+  styles: {
+    contextualMenu: [backgroundColorTokens, borderTokens],
+    button: [backgroundColorTokens, borderTokens],
```

Comment:
Should we add `textTokens` to customize the text of the button?

Reply:
I think I'd also expect to see something like 

`export interface MenuButtonTokens extends BackgroundColorTokens, BorderTokens, FontTokens;`

defined in MenuButton.types.ts and added to `MenuButtonType`

Reply:
Where will this interface be used? It's possible to customize it without it. Is there another way of customization? I mean except this one:
```
const StyledMenuButton = MenuButton.customize({
    button: {
      borderRadius: 4
    }
  });
```
Firstly I added it, but it had no impact on the component, I didn't use it anywhere

Reply:
I can't use textTokens in the button slot because it's not ITextProps. In Button component there's also icon and content properties in the style. Can I use it somehow in the button style property inside MenuButton?

Reply:
you should be able to add textTokens, foregroundColorTokens to the button slot. As Krystal mentioned, you should add tokens to the MenuButtonType interface in MenuButton.types

Reply:
Then you would be able to support `button: [backgroundColorTokens, borderTokens, textTokens, foregroundColorTokens]`

Reply:
I added, but I still have this error

Reply:
> Where will this interface be used?

The MenuButtonTokens interface will be passed to tokens in the MenuButtonType interface like so:

```
export interface MenuButtonType {
  props: MenuButtonProps;
  slotProps: MenuButtonSlotProps;
  state: MenuButtonState;
  tokens: MenuButtonTokens;
}
```

Looks like you've already hooked that up. I'm still not sure why textTokens can't be added to button here but when I was testing things locally, it seems that it's not needed in order for text tokens to go through.